spec/features/quotes/index_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe 'Returning Quotes' do
```

Comment:
The only thing that you are missing in your testing is to also display the keyword that was used in the search
```
As well as the keyword that's being used for the search
```